stuffListApi/server.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express'),
+    app = express(),
+    port = process.env.PORT || 8080,
+    mongoose = require('mongoose'),
+    Stuff = require('./api/models/stuffListModel'),
+    bodyParser = require('body-parser');
+
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/stuffListdb');
```

Comment:
Yeah, I was also getting same issue but I did not realize why. I was reading about that, and maybe here we can replace the connect code to something like this: 
    mongoose.connect(databaseUri, { useMongoClient: true })
        .then(() => {
            console.log('Mongodb connected at  ', databaseUri);
            return mongoose.connection;
        })
        .catch(err => debug('Connection error: ${err.message}'));